src/shared/services/configuration/map-markers.ts

```diff
@@ -17,6 +17,12 @@ export const markerIcon = L.icon({
   className: 'map-marker-select',
 })
 
+export const selectedMarkerIcon = L.icon({
+  iconUrl: '/assets/images/featureSelectedMarker.svg',
```

Comment:
better use kebab case, like most in assets/images

Reply:
I didn't add the marker file. So to avoid having many files changed (it's used on other places as well). I did not change it.